refactor/src/JSONEncoders.py

```diff
@@ -18,7 +18,7 @@ def default(self, o):
         return_str = ""
         if isinstance(o, np.ndarray):
             return_str = str(o)
-        elif isinstance(o, np.int64):
+        elif isinstance(o, int):
```

Comment:
Not needed. for basic types simply return str(o) or there is a risk they may be converted into an empty string.

Reply:
#87